### ISSUE TITLE
CI

### ISSUE THREAD

**Issue by terfn** [terfn/eecs-io-new#1]
Perhaps we could use [Travis](travis-ci.org) to test out each pr and make sure it doesn't break anything. 

@dimazest I think you came up with the idea before?

**Comment by dimazest** [comment]
Yeah, though I usually deploy things by myself as it faster, but it should not be hard to make it running on travis. Once Makefile is completed you need to run two commands:
```
make bootstrap
make deploy
```

have a look to https://bitbucket.org/dimazest/dima_qmul_homepage/src/ceb52924e669295ccc5e79a79367d60963d25dcf/Makefile?at=master&fileviewer=file-view-default
